released/discovery_center/mission_3252/provider.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    btp = {
+      source  = "SAP/btp"
+      version = "~> 1.4.0"
+    }
+  }
+}
+
+provider "btp" {
+  #cli_server_url = var.cli_server_url
```

Comment:
Please remove this variable or uncomment it and set a default value via a variable that points to the life landscape

Reply:
removed